src/app/event-external-root/event-external-root.controller.js

```diff
@@ -3,14 +3,29 @@
 angular.module('esn.calendar.event-consultation')
   .controller('CalEventExternalRootController', CalEventExternalRootController);
 
-function CalEventExternalRootController($scope, result) {
+function CalEventExternalRootController($rootScope, $location, result, calEventExternalRootService) {
   const self = this;
 
+  $rootScope.$on('UPDATE_ACTION_EXCAL', function(event, newLink) {
+    const url = new URL('http://localhost:8080' + newLink);
```

Comment:
I'm pretty sure http://localhost:8080/ is not the correct URL.

Reply:
the goal of this line is to create a fake url to extract jwt params but i am still working on it